openfisca_france_local/aides_jeunes_reform.py

```diff
@@ -0,0 +1,269 @@
+
+    comparison = operations[condition['operator']]
+
+    return comparison(individus_age, condition_age)
+
+
+def is_department_eligible(individu: Population, period: Period, condition):
+    depcom = individu.menage('depcom', period.first_month)
+    return sum([startswith(depcom, code.encode('UTF-8'))for code in condition['values']])
+
+
+def is_region_eligible(individu: Population, period: Period, condition):
+    eligibilite_region_table: dict = {
+        '01': 'guadeloupe_eligibilite_residence',
+        '02': 'martinique_eligibilite_residence',
+        '03': 'guyane_eligibilite_residence',
+        '04': 'reunion_eligibilite_residence',
+        '11': 'ile_de_france_eligibilite_residence',
+        '24': 'centre_val_de_loire_eligibilite_residence',
+        '27': 'bourgogne_franche_comte_eligibilite_residence',
+        '28': 'normandie_eligibilite_residence',
+        '32': 'hauts_de_france_eligibilite_residence',
+        '44': 'grand_est_eligibilite_residence',
+        '52': 'pays_de_la_loire_eligibilite_residence',
+        '53': 'bretagne_eligibilite_residence',
+        '75': 'nouvelle_aquitaine_eligibilite_residence',
+        '76': 'occitanie_eligibilite_residence',
+        '84': 'auvergne_rhone_alpes_eligibilite_residence',
+        '93': 'provence_alpes_cote_d_azur_eligibilite_residence',
+        '94': 'corse_eligibilite_residence',
+    }
+    return sum([individu.menage(eligibilite_region_table[code_region], period.first_month) for code_region in condition['values']])
+
+
+def is_regime_securite_sociale_eligible(individu: Population, period: Period, condition):
+    regime_securite_sociale = individu('regime_securite_sociale', period)
+    return sum([regime_securite_sociale == RegimeSecuriteSociale[regime] for regime in condition['includes']])
+
+
+def is_quotient_familial_eligible(individu: Population, period: Period, condition) -> np.array:
+
+    rfr = individu.foyer_fiscal('rfr', period.this_year)
+    nbptr = individu.foyer_fiscal('nbptr', period.this_year)
+    quotient_familial = rfr / nbptr
+
+    comparison = operations[condition['operator']]
+
+    return comparison(quotient_familial, condition['value'])
+
+
+def is_formation_sanitaire_social_eligible(individu: Population, period: Period, condition) -> np.array:
+    id_formation_sanitaire_social = GroupeSpecialitesFormation.groupe_330
+    id_formation_groupe = individu(
+        'groupe_specialites_formation', period.first_month)
+    return id_formation_groupe == id_formation_sanitaire_social
+
+
+def is_beneficiaire_rsa_eligible(individu: Population, period: Period, condition) -> np.array:
+    rsa = individu.famille('rsa', period)
+    return rsa > 0
+
+
+def is_annee_etude_eligible(individu: Population, period: Period, condition) -> np.array:
+    current_year = individu(
+        'annee_etude', period.first_month)
+    return sum([current_year == TypesClasse[value] for value in condition['values']])
```

Comment:
```suggestion
    return sum([current_year == TypesClasse[value] for value in condition['values']]) > 0
```

Pour éviter des effets de bord